pyuvdata/utils.py

```diff
@@ -817,16 +796,7 @@ def XYZ_from_LatLonAlt(latitude, longitude, altitude):
             "latitude, longitude and altitude must all have the same length"
         )
 
-    # see wikipedia geodetic_datum and Datum transformations of
-    # GPS positions PDF in docs/references folder
-    gps_n = gps_a / np.sqrt(1 - e_squared * np.sin(latitude) ** 2)
-    xyz = np.zeros((n_pts, 3))
-    xyz[:, 0] = (gps_n + altitude) * np.cos(latitude) * np.cos(longitude)
```

Comment:
I know it's a little outside the scope of this PR, but these rotation transformations are "almost magical" to me, in the sense that it's not obvious why we're doing this particular rotation. There is [this memo in the docs](https://github.com/RadioAstronomySoftwareGroup/pyuvdata/blob/master/docs/references/GPS.G1-X-00006_Datum_Transformations.pdf) that explains a little bit, but we may want to consider having some documentation that explains the difference between ECEF/rotECEF/ENU/EIEIO, etc. It's not worth doing here, and instead I'll make a separate issue about it.

Reply:
I've now made #845 to track this issue.